talent/job-descriptions/applications_eng_SF.md

```diff
@@ -0,0 +1,56 @@
+- At source{d} we are building the stack for the next generation of Machine Learning powered developer tools.
+
+- We have raised over eight million USD so far, and we are currently growing our team.
+
+- Our current monetization strategy is based on deploying our stack and tooling within large tech companies.
+
+## TEAM
+
+- Engineering consists of five different teams, that mimic the architecture of our product:
+
+- Applications (Scala, Go and Python and Frontend tools): Builds CLI/Web applications based on ML research.
+- Machine Learning (C++ and Python): R&D of machine learning on source code.
+- Data Retrieval (Scala and Go): Builds the technology that finds, fetches, stores and analyzes over +60M Git repositories.
+- Language Analysis (Go and another +15): Works on Babelfish, the universal code parsing server.
+- Infrastructure (Go and Python): Manages on-prem bare metal servers with Kubernetes and CoreOS.
+ 
+We really care about Open Source. Everything we develop is available for anyone to read, modify, and contribute.
+Our libraries and applications are available under the following GitHub organizations:
+
+- [bblfsh/dashboard](https://github.com/bblfsh/dashboard): a playground application for Babelfish server, turning code files into Universal Abstract Syntax Trees (UASTs). 
+- [src-d/kmcuda](https://github.com/src-d/kmcuda):  fast K-means and K-nearest neighbours on NVIDIA GPUs.  
+- [src-d/minhashcuda](https://github.com/src-d/minhashcuda):  fast Weighted MinHash on NVIDIA GPUs. It allows for fast nearly-duplicate repository detection: scan +17 git repositories in 30 minutes to find 1.5M duplicates (the results were published on data.world).  
+- [src-d/engine](https://github.com/src-d/engine):  a library for running scalable data retrieval pipelines that process any number of Git repositories for source code analysis.  
+- [src-d/hercules](https://github.com/src-d/hercules):  analysis and visualization of a Git repository history. 
+- [src-d/gemini](https://github.com/src-d/gemini): a duplicate finder for source code in git repositories.
+
+## ROLE
+
+This position is for a Senior Engineer on the Applications team, based in SF or remote. 
+
+- You will be expected to have a deep understanding and previous experience in developer tooling on a large scale. 
```

Comment:
I would suggest being more specific by adding something like  `2+ years of previous experience working on dev tooling...`